presto-postgresql/src/main/java/com/facebook/presto/plugin/postgresql/PostgreSqlClient.java

```diff
@@ -105,17 +106,24 @@ protected ResultSet getTables(Connection connection, Optional<String> schemaName
     }
 
     @Override
-    protected String toSqlType(Type type)
+    public WriteMapping toWriteMapping(Type type)
     {
         if (VARBINARY.equals(type)) {
-            return "bytea";
+            return sliceMapping("bytea", (SliceWriteFunction) varbinaryColumnMapping().getWriteFunction());
         }
-
-        return super.toSqlType(type);
+        else if (type.getTypeSignature().getBase().equals(StandardTypes.JSON)) {
+            return sliceMapping("jsonb", ((statement, index, value) -> {
```

Comment:
why is this `jsonb`? Are we able to convert to the binary form for postgres? I think it should be `json`

Reply:
This also needs tests